cmd/tracer/main.go

```diff
@@ -0,0 +1,61 @@
+package main
```

Comment:
so this is just a test binary for generating a trace? personally i prefer to just leave code like this out of the repo (if we need examples let's put them in our godoc comments) but that's not a strong preference